documentation/src/main/asciidoc/userguide/index.adoc

```diff
@@ -9,6 +10,7 @@ include::{shared-attributes-dir}/renderer-attributes.adoc[]
 :toc2:
 :toclevels: 3
 :sectanchors:
+:version-selector-enabled: true
```

Comment:
This is a nice features, perhaps we should also have it on other docs? Migration guide and what's new seems like the obvious candidates, perhaps even the introduction guide (though that didn't exist in earlier versions, I wonder what happens then).

Reply:
+1 it would be nice to have it 🙂  . Though we'll have to figure out a better way to create links for the versions...
Right now, the way that selector works is that it just tries to replace the version in the path, so for these other guides, we may end up with a lot of 404 😖 

but Yoann has this idea: https://github.com/hibernate/hibernate.org/pull/285 maybe with that we can do better on the other docs 🤞🏻